apps/api/app/services/workflow/context_extractor.py

```
@@ -0,0 +1,273 @@
+        "get_trigger_schema",
+        "list_workflows",
+        "get_workflow",
+        "execute_workflow",
+        "finalize_workflow",
+    }
+
+    @classmethod
+    async def extract_from_thread(
+        cls,
+        thread_id: str,
+        user_id: str,
```

Ruff (0.14.14)

[warning] Unused class method argument: `user_id`

(ARG003)